02-props-state-component-architecture/props/Props Homework/src/Pokecard.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react'
+import './Pokecard.css'
+
+class Pokecard extends Component {
+	render(){
```

Comment:
in the future this can be a stateless functional component since all you do is `render`, but I don't think we taught you that before this exercise, so no worries.